ggml.c

```diff
@@ -161,6 +161,8 @@ inline static void* ggml_aligned_malloc(size_t size) {
 #endif
 #elif defined(GGML_USE_OPENBLAS)
 #include <cblas.h>
+#elif defined(GGML_USE_KOMPUTE)
+#include "ggml-vulkan.h"
```

Comment:
Ideally, you don't want to include `ggml-vulkan.h` in `ggml.c` similar to the Metal implementation